dataplanes/vpp/pkg/nsmutils/nsm-utils.go

```diff
@@ -113,3 +86,14 @@ func ipv4prefixlength(value string) error {
 	}
 	return nil
 }
+
+func Empty(value string) error {
+	return nil
+}
+
+func Bool(value string) error {
+	if _, err := strconv.ParseBool(value); err != nil {
+		return err
+	}
+	return nil
+}
```

Comment:
Why not just use strconv.ParseBool(value) directly?

Reply:
Ah... I get it now, you are creating these funcs to use as fields in the validator.  Got it :)